doc/release-notes/master.txt

```diff
@@ -117,6 +117,12 @@ Vital
    moved into vital/io, and the KLV conversion functions have moved into
    vital/klv.
 
+ * Refactor timestamp types for time and frame number. Standardized on
+   the base kwiver::vital types for frame_id and time. Renamed time_t
+   to time_us_t to provide some distance from std::time_t and make the
+   units apparent. Removed the type names from the timestamp class.
+
```

Comment:
The second blank line here seems out of place; we only had one, previously, and I don't see where we have two elsewhere in this file...